shared/util/codeql/util/test/InlineExpectationsTest.qll

```diff
@@ -165,14 +156,19 @@ module Make<InlineExpectationsTestSig Impl> {
      * A failure will still arise if there is an annotation that does not match any results, but not vice versa.
      * Override this predicate to specify optional results.
      */
-    predicate hasOptionalResult(Location location, string element, string tag, string value) {
-      none()
-    }
+    predicate hasOptionalResult(Impl::Location location, string element, string tag, string value);
```

Comment:
This could have a default `none()` implementation.

Reply:
Can we do that now? Note that this refers to part of the signature: `Impl::Location`.

Reply:
I see we have an internal PR open that'll likely help here. We should probably hold this off until that has landed in a release.

Reply:
Ah yes, you are right, we can't do that just yet.

Reply:
@hvitved `none()` implementation added. I assume we still need to wait with this PR until support for referencing other parts of the module signature in defaults is available in a release (which it isn't yet, should be 2.13.4).

Reply:
I just doubled checked it seems that the default here was not working for some other reason when I last tried this. Testing this with CodeQL 2.13.1 this seems to work out-of-the-box.

Reply:
The `Compile all queries using the latest stable CodeQL CLI` check should be enough to ensure that it works with the latest release.